habitat_baselines/config/default.py

```diff
@@ -55,6 +55,7 @@
 # REINFORCEMENT LEARNING (RL) ENVIRONMENT CONFIG
 # -----------------------------------------------------------------------------
 _C.RL = CN()
+_C.RL.fp16_mode = "off"
```

Comment:
What's the stylgeuide for what should be all CAPS or not. shouldn't it FP16_mode?